djhubspot/client.py

```diff
@@ -69,56 +71,63 @@ def __init__(self, hubspot_api_key=None):
         """
         self.hubspot_api_key = hubspot_api_key or settings.HUBSPOT_API_KEY
 
-    def _get_associations_client(self):
+    # TODO: We could simplify the following lines by using @property instead of getters.
+
+    def get_associations_client(self):
```

Comment:
Why make them public?

Reply:
Because sometimes the `dj-hubspot` implementation is not enough for my needs and I find frustrating to have to call a private method in order to be able to retrieve the client I need.

Reply:
:ok: